third_party/libadblockplus/grit/resources/elemhide_for_selector.jst

```diff
@@ -0,0 +1,13 @@
+// {{filename_with_query}} and {{url}} will be replaced with actual variable values
+
+function elemhideForSelector(selector) {
```

Comment:
Yes and no, it's definitely sub-optimal because in this case [last declared](https://www.ecma-international.org/ecma-262/#sec-functiondeclarationinstantiation) function is used, so we have to compile and use a new function each time.

Reply:
then we need to do or not to do this for both scripts. And since the first one will come from `core` module we will have to spread this logics to core or wrap it into one more script that just checked if the function from core is already defined or define it otherwise. We're going too deeply in the details imho